### PR TITLE
bridge-scrow -> tests.rs

### PR DESCRIPTION
**Pull request by doradelta**

### CODE REVIEW

File: solana/bridge-escrow/programs/bridge-escrow/Cargo.toml [hunk 1]
```diff
@@ -18,6 +18,7 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
+anyhow.workspace = true
```
Comment (mina86) [inline solana/bridge-escrow/programs/bridge-escrow/Cargo.toml:21]: First of, this is only used in tests so should be in `[dev-dependencies]`. Second of, why not just `unwrap` in tests? We really don’t need error handling in a test.

File: solana/bridge-escrow/programs/bridge-escrow/src/lib.rs [hunk 2]
```diff
@@ -98,16 +92,23 @@ pub mod bridge_escrow {
             },
         );
 
-        let memo = "{\"forward\":{\"receiver\":\"0x4c22af5da4a849a8f39be00eb1b44676ac5c9060\",\"port\":\"transfer\",\"channel\":\"channel-52\",\"timeout\":600000000000000,\"next\":{\"memo\":\"my-custom-msg\"}}}".to_string();
+        let memo = "{\"forward\":{\"receiver\":\"\
+                    0x4c22af5da4a849a8f39be00eb1b44676ac5c9060\",\"port\":\"\
+                    transfer\",\"channel\":\"channel-52\",\"timeout\":\
+                    600000000000000,\"next\":{\"memo\":\"my-custom-msg\"}}}"
+            .to_string();
         let memo = memo.replace("my-custom-msg", &my_custom_memo);
```
Comment (mina86) [inline solana/bridge-escrow/programs/bridge-escrow/src/lib.rs:100]: Why use replace? Use format instead. And also, how about formatting it better:

        let memo = format!(
            "{\"forward\":{\
                "receiver\":\"0x4c22af5da4a849a8f39be00eb1b44676ac5c9060\",\
                \"port\":\"transfer\",\
                \"channel\":\"channel-52\",
                \"timeout\":600000000000000,\
                \"next\":{\"memo\":\"{},{},{},{}\"}\
            }}",
            source_account.key(),
            destination_account.key(),
            authority.key(),
            token_program.key(),
        );

Add `[rustfmt::skip]` if fmt breaks the formatting.

File: solana/bridge-escrow/programs/bridge-escrow/src/lib.rs [hunk 3]
```diff
@@ -98,16 +92,23 @@ pub mod bridge_escrow {
...
 
         // MsgTransfer
         let msg = MsgTransfer {
             port_id_on_a: PortId::from_str("transfer").unwrap(),
```
Comment (mina86) [inline solana/bridge-escrow/programs/bridge-escrow/src/lib.rs:104]:
```suggestion
            port_id_on_a: PortId::transfer(),
```

**Comment by mina86** [comment]
Also, add a proper PR title and description

**Comment by dhruvja** [comment]
Merging this into `fast-bridge` branch and opening a new PR.

**Pull request merged**